linera-core/src/client/mod.rs

```diff
@@ -157,6 +157,9 @@ where
     message_policy: MessagePolicy,
     /// Whether to block on cross-chain message delivery.
     cross_chain_message_delivery: CrossChainMessageDelivery,
+    /// Grace period for communicate_with_quorum as a fraction of time taken to reach quorum.
+    /// If None, uses the default value.
```

Comment:
```suggestion
    /// Grace period for `communicate_with_quorum` as a fraction of time taken to reach quorum.
    /// If `None`, uses the default value.
```

Or maybe we should not mention the name of the method. "An additional delay, after reaching a quorum, to wait for additional validator signatures."?